README.md

```diff
@@ -6,20 +6,35 @@ A set of MapReduce programs to process brain images
 ##Histogram generation
 
 Program to generate the histogram of given nifti image. <br/>
-Note: Data binning is not currently implemented and thus bin size is always 1.
  
 ###Java implementation
 To run:<br/>
 <code>hadoop jar GetHistogram.jar GetHistogram <input file> <output folder></code>
 
 Dependencies: niftijio - https://github.com/cabeen/niftijio
 
+Note: Data binning is not currently implemented for java thus bin size is always 1.
 
 ###Python implementation
 
 To run:<br/>
-<code>hadoop jar < hadoop streaming jar path > -input < input filename > -output < output directory > -mapper "python VoxelMapper.py" -reducer "python VoxelCountReducer.py"</code>
-
+<code>hadoop jar < hadoop streaming jar path > -input < input filename > -output < output directory > -mapper "python VoxelMapper.py [arguments]" -reducer "python VoxelCountReducer.py"</code>
+
+Possible input arguments for the mapper:
+<ul>
+<li>-nb --num_bins : the number of bins contained within the histogram. This value must be in int format and must be followed by a float representing the maximum value of the dataset. Optionally, a minimum value can be inputted. If a minimum value is not supplied, a value of 0 is assumed to be the minimum.
```

Comment:
I guess you need the max value to avoid having to do a 2nd pass on the data. That's fine, but maybe you can create an issue to make the code determine the min and max.